lib/src/core/comms/message_chunk.rs

```diff
@@ -7,6 +7,7 @@
 
 use std::io::{Cursor, Read, Write};
 
+use crate::puffin::types::OpcuaProtocolTypes;
```

Comment:
[globally] Please use absolute path instead of importing with `use` to minimize potential namespace clash and number of modified LoC.